app/views/manage/show.html.erb

```diff
@@ -30,59 +30,72 @@
 </div>
 
 <%= render "govuk_publishing_components/components/heading", {
-  text: t("account.manage.privacy.heading"),
+  text: t("account.manage.cookies.title"),
   heading_level: 2,
   font_size: "m",
   margin_bottom: 4,
 } %>
 
-<div class="accounts-summary-list accounts-summary-list--wide">
-  <dl class="govuk-summary-list">
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <p class="govuk-body accounts-summary-list__key-heading"><%= t("account.manage.privacy.cookies_question") %></p>
-        <p class="govuk-body accounts-summary-list__key-description"><%= t("account.manage.privacy.cookies_description") %></p>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= current_user.cookie_consent == true ? t("general.yes") : t("general.no") %>
-      </dd>
-      <dd class="govuk-summary-list__actions">
-        <a href="<%= edit_user_consent_cookie_path %>"
-          class="govuk-link"
-          title="<%= t("general.change") %> <%= t("account.manage.privacy.cookies_link_extra") %>"
-          data-module = "gem-track-click"
-          data-track-category = "account-manage"
-          data-track-action = "manage-account"
-          data-track-label = "personal-information-cookies">
-          <%= t("general.change") %>
-          <span class="govuk-visually-hidden"><%= t("account.manage.privacy.cookies_link_extra") %></span>
-        </a>
-      </dd>
-    </div>
+<p class="govuk-body"><%= t("account.manage.cookies.description") %></p>
 
-    <div class="govuk-summary-list__row">
-      <dt class="govuk-summary-list__key">
-        <p class="govuk-body accounts-summary-list__key-heading"><%= t("account.manage.privacy.email_question") %></p>
-        <p class="govuk-body accounts-summary-list__key-description"><%= t("account.manage.privacy.email_description") %></p>
-      </dt>
-      <dd class="govuk-summary-list__value">
-        <%= current_user.feedback_consent == true ? t("general.yes") : t("general.no") %>
-      </dd>
-      <dd class="govuk-summary-list__actions">
-        <a href="<%= edit_user_consent_feedback_path %>"
-          class="govuk-link"
-          title="<%= t("general.change") %> <%= t("account.manage.privacy.email_link_extra") %>"
-          data-module = "gem-track-click"
-          data-track-category = "account-manage"
-          data-track-action = "manage-account"
-          data-track-label = "personal-information-feedback">
-          <%= t("general.change") %>
-          <span class="govuk-visually-hidden"><%= t("account.manage.privacy.email_link_extra") %></span>
-        </a>
-      </dd>
-    </div>
-  </dl>
-</div>
+<% cookie_link_text = capture do %>
+  <%= t("general.change") %> <span class="govuk-visually-hidden"><%= t("account.manage.cookies.link_extra") %></span>
```

Comment:
Just a thought (which I probably should have put on the Summary links component PR as I was reviewing, and not on this one but alas):

The current approach for overriding the default hidden text is using `link_text_no_enhance: true` in combination of passing through the hidden text like **Visible text `<span class="govuk-visually-hidden">`hidden text`</span>`**.

I wonder if we could we have achieved the same by passing the hidden text as a separate `link_hidden_text` parameter instead of passing it through with the `link_text`. And then handling the logic of whether `<span class="govuk-visually-hidden">...</span>` should use `link_hidden_text`, or the default hidden text, or no text at all in the component itself.

The whole capture with the hidden span thing feels a bit dirty is all, but I suppose we do it in lots of other places too.
Ignore if it doesn't make sense, I think I'm starting to ramble a bit.

Reply:
That makes sense, and I considered that approach, but I think this way is more flexible. For example, it allows you to have no hidden text, which might be required - the visible link text might already be descriptive enough.